corehq/apps/fixtures/static/fixtures/js/built.js

```diff
@@ -0,0 +1,16 @@
+/* globals define */
+define("fixtures/js/built", [
+    "reports/js/filters",
+    "reports/js/standard_hq_report",
+    "reports/js/config.dataTables.bootstrap",
+], function(
+    filters,
+    standardHQReportModule,
+    datatablesConfig
+){
+    return {
+        filters: filters,
+        standardHQReportModule: standardHQReportModule,
+        datatablesConfig: datatablesConfig,
```

Comment:
holy unfortunate DRY violations! not sure there's anything to be done about it, just ranting